NineChroniclesUtilBackend/Repositories/ArenaRankingRespository.cs

```diff
@@ -41,21 +41,18 @@ public async Task<List<ArenaRanking>> GetRanking(long limit, long offset)
     {
         var pipelines = new[]
         {
-            @"{ $sort: { 'Score.Score': -1 } }",
-            @"{ $group: { _id: null, docs: { $push: '$$ROOT' } } }",
-            @"{ $unwind: { path: '$docs', includeArrayIndex: 'Rank' } }",
+            @"{ $setWindowFields: { partitionBy: '', sortBy: { 'Score.Score': -1 }, output: { Rank: { $rank: {} } } } }",
             $@"{{ $skip: {offset} }}",
             $@"{{ $limit: {limit} }}",
-            @"{ $replaceRoot: { newRoot: { $mergeObjects: [ '$docs', { Rank: '$Rank' } ] } } }",
             @"{ $lookup: { from: 'avatars', localField: 'AvatarAddress', foreignField: 'Avatar.address', as: 'Avatar' } }",
             @"{ $unwind: { path: '$Avatar', preserveNullAndEmptyArrays: true } }",
+            $@"{{ $unset: ['Avatar.Avatar.inventory', 'Avatar.Avatar.mailBox', 'Avatar.Avatar.stageMap', 'Avatar.Avatar.monsterMap', 'Avatar.Avatar.itemMap', 'Avatar.Avatar.eventMap'] }}",
```

Comment:
It seems it doesn't use template here 👀 
```suggestion
            @"{ $unset: ['Avatar.Avatar.inventory', 'Avatar.Avatar.mailBox', 'Avatar.Avatar.stageMap', 'Avatar.Avatar.monsterMap', 'Avatar.Avatar.itemMap', 'Avatar.Avatar.eventMap'] }",
```